access/grpc/client.go

```diff
@@ -103,6 +113,11 @@ type Client struct {
 	grpc *BaseClient
 }
 
+// RPCClient returns the underlying gRPC client.
+func (c *Client) RPCClient() RPCClient {
```

Comment:
this isn't strictly needed here, but it provides some flexibility for users in the future to break out into a raw grpc request when the sdk doesn't expose some needed/new functionality